features/keychain/module/keychain.js

```diff
@@ -133,6 +171,7 @@ export class Keychain {
   }
 
   clone() {
+    assert(!this.#lockedPrivateKeys, 'private keys are locked')
```

Comment:
clone doesn't copy private keys, it just creates a new Keychain with the same `legacyPrivToPub` functions
```suggestion
```

Reply:
6cee39e4e283049c120cb7454d582e5d0d2025d2